internal/lib/avi_api.go

```diff
@@ -234,5 +234,10 @@ func NewAviRestClientWithToken(api_ep string, username string, authToken string)
 		utils.AviLog.Warnf("NewAviClient returned err %v", err)
 		return nil
 	}
+	controllerVersion := utils.CtrlVersion
```

Comment:
does the controller version need to be set to max supported version here if specified controller version is greater than max supported version?